hw/top_darjeeling/chip_darjeeling_asic.core

```diff
@@ -0,0 +1,84 @@
+CAPI=2:
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:systems:chip_darjeeling_asic:0.1"
+description: "Earl Grey chip level"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:systems:top_darjeeling:0.1
+      - lowrisc:systems:top_darjeeling_pkg
+      - lowrisc:systems:top_darjeeling_padring
+      - "fileset_partner ? (partner:systems:ast)"
```

Comment:
The ast is a good example of a clumsy VLNV: the core should be "opentitan: earlgrey_systems:ast" and "opentitan:darjeeling_systems:ast". Heck, the "systems" part is also silly, and could just be "ip". The problem is that the current naming doesn't work well with real multitop: "partner" is a sort of singleton, and "systems" is too broad. I imagine we will have flags to name the top.